.github/workflows/build_worker_images.yml

```diff
@@ -31,6 +31,12 @@ jobs:
           restore-keys: |
             pip-
       - run: pip install -r requirements.txt
+      - run: | 
+          sudo swapoff -a
+          sudo rm -f /swapfile
+          sudo apt clean
+          docker rmi $(docker image ls -aq)
```

Comment:
I think we should also add

`sudo rm -rf /opt/ghc`

(from https://github.com/codalab/codalab-worksheets/blob/8857244823146c0fe0169fabdc29c4d698d84667/.github/workflows/test.yml#L103-L106)